ee/control/consumers/notificationconsumer/notify_consumer.go

```diff
@@ -253,16 +219,8 @@ func (nc *NotificationConsumer) cleanup() {
 	}
 
 	// Delete all old keys
-	if err := nc.db.Update(func(tx *bbolt.Tx) error {
-		for _, k := range keysToDelete {
-			if err := tx.Bucket([]byte(osquery.SentNotificationsBucket)).Delete(k); err != nil {
-				// Log, but don't error, since we might be able to delete some others
-				level.Debug(nc.logger).Log("msg", "could not delete old notification from bucket", "key", string(k), "err", err)
-			}
-		}
-
-		return nil
-	}); err != nil {
-		level.Debug(nc.logger).Log("msg", "could not delete old notifications from bucket", "err", err)
+	if err := nc.store.Delete(keysToDelete...); err != nil {
+		// Log, but don't error, since we might be able to delete some others
+		level.Error(nc.logger).Log("msg", "could not delete old notifications from bucket", "err", err)
```

Comment:
Since this isn't looping over the keys anymore, I think we could return the error now

Reply:
I could have `cleanup()` return an error, but since it's run periodically I think the error will just get logged like this (either here or in `runCleanup`). I suppose the comment here is inaccurate now though...

Reply:
I'm fine with just removing the comment, then!

Reply:
Cool, I'll adjust it in the next PR!